crates/gitbutler-core/src/virtual_branches/state.rs

```diff
@@ -151,10 +153,11 @@ impl VirtualBranchesHandle {
 
 fn write<P: AsRef<Path>>(file_path: P, virtual_branches: &VirtualBranches) -> anyhow::Result<()> {
     let contents = toml::to_string(&virtual_branches)?;
-    let temp_file = tempfile::NamedTempFile::new_in(file_path.as_ref().parent().unwrap())?;
-    let (mut file, temp_path) = temp_file.keep()?;
-    file.write_all(contents.as_bytes())?;
-    drop(file);
-    std::fs::rename(temp_path, file_path.as_ref())?;
-    Ok(())
+    let mut temp_file = gix::tempfile::new(
+        file_path.as_ref().parent().unwrap(),
```

Comment:
is it safe to do "unwrap" here, @Byron?

Reply:
It's best avoided, but the only reason this fails is if the path is `/` I think. So in practice, it's safe, but probably best to prefer `.expect("why")` here.